### PR TITLE
Update harvard-stellenbosch-university.csl

### PR DESCRIPTION
**Pull request by Yusufras**
All the authors must show when cited for the first time. After that it is et.al.

### CODE REVIEW

**Comment by Yusufras** [comment]
All the authors must show when cited for the first time. After that it is et. al.

File: harvard-stellenbosch-university.csl [hunk 1]
```diff
@@ -63,7 +63,7 @@
   </macro>
   <macro name="author-short">
     <names variable="author">
-      <name form="short" and="symbol" delimiter-precedes-last="never" et-al-min="6" et-al-use-first="5" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize-with="." sort-separator=" "/>
+      <name form="short" and="symbol" delimiter-precedes-last="never" et-al-min="10" et-al-use-first="5" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize-with="." sort-separator=" "/>
```
Comment (rmzelle) [inline harvard-stellenbosch-university.csl]: @Yusufras, what did you base this on?

I could only find:

> For first reference in text:
Coherence is an aspect of discourse competency that refers to the sequence of continuity of ideas in written or spoken texts (Davies, Brown, Elder, Hill, Lumley & McNamara, 1999:25).

> For all subsequent references in text:
Coherence is an aspect of discourse competency that refers to the sequence of continuity of ideas in written or spoken texts (Davies et al., 1999:25).

at http://libguides.sun.ac.za/content.php?pid=344501&sid=2970868

**Comment by Yusufras** [comment]
We need to have all the authors listed in the intext citation for the first reference. All subsequent references then needs to be et al. for the same authors.

Our examples states 6 authors, but what if there are more than 6?

Accroding to Harvard referencing convention at Stellenbosch University: … (Davies, Brown, Elder, Hill, Lumley & McNamara, 1999:25).
Mendeley does … (Davies, Brown, Elder, Hill, Lumley, et al., 1999:25)       … it cuts off the other authors names at a point and only show them in the bibliography or reference list

For all subsequent references in text, Mendeley does it correctly:
     (Davies et al., 1999:25).

Reference list:
     Davies, A., Brown, A. , Elder, C. , Hill, K., Lumley, T. & McNamara, T. 1999. Dictionary of language testing. Cambridge: Cambridge University Press.

**Comment by rmzelle** [comment]
@Yusufras, I understand that the "et-al" settings in the CSL style currently don't match the example I highlighted. However, I was wondering if you were aware of any more examples that explain at exact what number of authors "et al." should be used? While `et-al-min="10" et-al-use-first="5"` would fix the example, so would removing the attributes altogether.

**Comment by Yusufras** [comment]
We do not have an example but I think that the cut off should be 10.

Is the style updated with the latest changes?

Vriendelike groete / Kind regards
Yusuf Ras

Fakulteitsbibliotekaris AgriWetenskappe / Faculty Librarian AgriSciences
Biblioteek- & Inligtingsdiens / Library & Information Service
Universiteit Stellenbosch University  | map<https://www.google.co.za/maps/place/JS+Gericke+Library/@-33.932975,18.8640494,377m/data=!3m1!1e3!4m2!3m1!1s0x1dcdb25d82940c29:0x80f6dcd585958374> | Tel: +27 (021) 808 4345 | yr@sun.ac.za<mailto:yr@sun.ac.za>

**Comment by rmzelle** [comment]
Ah, I didn't realize you were a librarian at Stellenbosch. In that case, I'll trust that you have this right :).

It would be good if your website (http://libguides.sun.ac.za/content.php?pid=344501&sid=2970868) would clarify what the exact et-al abbreviation rules should be, though.

**Comment by Yusufras** [comment]
I only changed the et al., to 10. Everything else is still the same. I do not understand the report. Any help please.

Vriendelike groete / Kind regards
Yusuf Ras

Fakulteitsbibliotekaris AgriWetenskappe / Faculty Librarian AgriSciences
Biblioteek- & Inligtingsdiens / Library & Information Service
Universiteit Stellenbosch University  | map<https://www.google.co.za/maps/place/JS+Gericke+Library/@-33.932975,18.8640494,377m/data=!3m1!1e3!4m2!3m1!1s0x1dcdb25d82940c29:0x80f6dcd585958374> | Tel: +27 (021) 808 4345 | yr@sun.ac.za<mailto:yr@sun.ac.za>

**Comment by rmzelle** [comment]
(just ignore the last error report; it happened because I had to reopen the pull request, which triggered a Travis build, and I merged the pull request before that build could successfully run)

**Pull request merged**